RussianFilter/sections/specific.txt

```diff
@@ -35,6 +35,7 @@ oka.fm##.rek-module
 ||cdn2-thevladarcompany.netdna-ssl.com/ru/wp-content/uploads/sites/4/2018/10/GN_Takeover.jpg
 pickpoint.ru##.bn-index-ad1
 mangalib.me##.ads
+||gyfwz.com^$domain=xn--e1afprfv.tv
```

Comment:
better move to adservers.txt with `third-party` instead of `domain=`

Reply:
yes

Reply:
moved to adservers.txt